src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SingleSpaceSeparatorCheck.java

```diff
@@ -77,6 +87,14 @@
  *   &lt;property name=&quot;validateComments&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * void fun1() {}    //violation, 4 whitespaces before the comment starts
+ * void fun2() {  /*violation here, 2 whitespaces before the comment starts
+ * OK here*&#47; return;
+ * }
+ * /*violation, 2 white spaces after the comment ends*&#47;  int a;
```

Comment:
please add example with javadoc comment

Reply:
So i should add a piece of code right on the line where a doc comment starts/ends, you mean?
Because the config will not work for any code inside the doc comment and if a piece of code is not on the same line where a doc comment starts/ends.

Reply:
I mean that here it is usual block comment, but it would be nice to have javadoc example